aiidalab/app.py

```diff
@@ -648,6 +699,20 @@ def _default_detached(self):
     def _default_busy(self):  # pylint: disable=no-self-use
         return False
 
+    @traitlets.validate("version_to_install")
+    def _validate_version_to_install(self, proposal):
```

Comment:
This function is not used anywhere, correct?

Reply:
It is called every time one wants to update the trait. And it does validate the trait. See in the tests:

```python
    # The version `v22.11.0` is incompatible while `v22.11.1` is compatible
    with pytest.raises(traitlets.TraitError):
        app.version_to_install = "v22.11.0"
    app.version_to_install = "v22.11.1"
```

Reply:
Nice solution, thanks! :+1:

Reply:
Interesting! Thanks for clearing up.